Formula/qpid-proton.rb

```diff
@@ -0,0 +1,47 @@
+class QpidProton < Formula
+  desc "High-performance, lightweight AMQP 1.0 messaging library."
+  homepage "https://qpid.apache.org/proton/"
+  url "https://www-us.apache.org/dist/qpid/proton/0.17.0/qpid-proton-0.17.0.tar.gz"
+  sha256 "6ffd26d3d0e495bfdb5d9fefc5349954e6105ea18cc4bb191161d27742c5a01a"
+  head "git://git.apache.org/qpid-proton.git"
+
+  depends_on "cmake" => :build
+  depends_on "libuv" if build.head?
+  depends_on "openssl"
+
+  def install
+    # Do not build language bindings
+    args = %w[CPP GO JAVASCRIPT PERL PHP PYTHON RUBY].map { |l| "-DBUILD_#{l}=OFF" }
```

Comment:
I'd probably tweak it like this:
```ruby
  args = std_cmake_args + %w[
    -DBUILD_CPP=OFF
    -DBUILD_GO=OFF
    -DBUILD_JAVASCRIPT=OFF
    -DBUILD_PERL=OFF
    -DBUILD_PHP=OFF
    -DBUILD_PYTHON=OFF
    -DBUILD_RUBY=OFF
  ]
  # Enforce installation into lib/ instead of lib64/
  args << "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=#{lib}"
```
Then you can just use:
```ruby
system "cmake", ".", *args
```